examples/forecasting/advanced_forecasting_example.py

```diff
@@ -0,0 +1,70 @@
+""" Example script for forecasting, demonstrating more advanced usage of fit().
+    Note that all settings demonstrated here are just chosen for demonstration purposes (to minimize runtime), and do not represent wise choices to use in practice.
+"""
+from autogluon.forecasting import ForecastingPredictor
+from autogluon.forecasting import TabularDataset
+import autogluon.core as ag
+
+train_data = TabularDataset("https://autogluon.s3-us-west-2.amazonaws.com/datasets/CovidTimeSeries/train.csv")
+test_data = TabularDataset("https://autogluon.s3-us-west-2.amazonaws.com/datasets/CovidTimeSeries/test.csv")
+static_features = TabularDataset("https://autogluon.s3-us-west-2.amazonaws.com/datasets/CovidTimeSeries"
+                                 "/toy_static_features.csv")
+# change this to specify eval metric, one of ["MASE", "MAPE", "sMAPE", "mean_wQuantileLoss"]
+eval_metric = "mean_wQuantileLoss"
+
+prediction_length = 19
+path = "hpo_models"
+
+predictor = ForecastingPredictor(path=path, eval_metric=eval_metric).fit(train_data,
+                                                                         prediction_length,
+                                                                         static_features=static_features,
+                                                                         index_column="name",
+                                                                         target_column="ConfirmedCases",
+                                                                         time_column="Date",
+                                                                         # hyperparameter_tune_kwargs={
```

Comment:
> why are these commented?

`hyperparameter_tune_kwargs` can either take a string as input to use default settings or it can take a dictionary to apply customized settings. A problem is that the customized setting for now must contain the keyword `scheduler` and `searcher`, maybe this can be optimized so that if some of the keywords are not provided we will give it a default value.

Reply:
I'll keep them for now as it shows there are both alternatives